internal/wasm/jit/arch_amd64.go

```diff
@@ -0,0 +1,62 @@
+package jit
+
+import (
+	"fmt"
+
+	asm "github.com/twitchyliquid64/golang-asm"
+	"github.com/twitchyliquid64/golang-asm/obj/x86"
+
+	wasm "github.com/tetratelabs/wazero/internal/wasm"
+	"github.com/tetratelabs/wazero/internal/wazeroir"
+)
+
+func init() {
```

Comment:
helps clarify when looking at files as not always notice the file name.

```suggestion
// init initializes variables for the amd64 architecture
func init() {
```